spring-cloud-function-adapters/spring-cloud-function-adapter-gcp/pom.xml

```diff
@@ -36,6 +36,12 @@
 			<groupId>org.springframework.cloud</groupId>
 			<artifactId>spring-cloud-function-context</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>org.json</groupId>
+			<artifactId>json</artifactId>
```

Comment:
@olegz Is this avoidable?

Reply:
yes, but not yet. I have a separate issue opened for that